README.md

```diff
@@ -9,7 +9,7 @@ This is KUKSA.val, the KUKSA **V**ehicle **A**bstration **L**ayer.
 
 KUKSA.val provides a [Genivi VSS data model](https://github.com/GENIVI/vehicle_signal_specification) describing data in a vehicle. This data is provided to applications using a variant based on the W3C VISS Interface. KUKSA.val supports VISS V1 https://www.w3.org/TR/vehicle-information-service/ and extensions as well as parts of the upcomming VISS2 standard (https://raw.githack.com/w3c/automotive/gh-pages/spec/Gen2_Core.html, https://raw.githack.com/w3c/automotive/gh-pages/spec/Gen2_Transport.html), that are applicable to in-vehicle VSS servers.
```

Comment:
see #127 the VISS links here are broken, @SebastianSchildt do you have another link?

Reply:
We had those links, becasue the VISS repo did not provide the, but that is repaired now: https://github.com/w3c/automotive/issues/371

So we can either link to their repo https://github.com/w3c/automotive (but that includes more tha just VISS standards), or - maybe better -  recycle the current  links from their Readme.
Which is 
Gen2 Core: https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Core.html
Gen2 Transport: https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Transport.html

Reply:
```suggestion
KUKSA.val provides a [Genivi VSS data model](https://github.com/GENIVI/vehicle_signal_specification) describing data in a vehicle. This data is provided to applications using a variant based on the W3C VISS Interface. KUKSA.val supports VISS V1 https://www.w3.org/TR/vehicle-information-service/ and extensions as well as parts of the upcomming VISS2 standard ([Gen2 Core](https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Core.html), [Gen2 Transport](https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Transport.html)), that are applicable to in-vehicle VSS servers.
```